run--01a--prep-geo-data.Rmd

```diff
@@ -533,17 +664,101 @@ if (exists("my_aux_geo")) {
              County = case_when(max(in_aux_flag) == 0 ~ County,
                                 in_aux_flag      == 0 ~ paste(County, glue("(Non-{my_aux_overall_name})")),
                                 in_aux_flag      == 1 ~ paste(County, glue("({my_aux_overall_name})"))),
-             pct_area = ifelse(is.na(pct_area), 1, pct_area))
+             pct_area = ifelse(is.na(pct_area), 1, pct_area)) %>% 
+      mutate(County = ifelse(County %like% "DuPage", "DuPage", County))
```

Comment:
Not a fan of this being merged to `main`. What's the exact problem here, and is there perhaps a way to generalize the solution?